opentracing/span.py

```diff
@@ -208,7 +203,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                 'python.exception.val': exc_val,
                 'python.exception.tb': exc_tb,
                 })
-        self.finish()
+        self.deactivate()
```

Comment:
Probably this can be kept as a `pass` statement as in other parts. The actual code/logic would go in `basictracer` (not an important part though).